File: server-scala-play-server/app/model/LineItem.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for LineItem.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class LineItem(
  description: Option[String],
  item: Option[String],
  pricePerItem: Option[Int],
  total: Option[Int],
  tax: Option[Int],
  quantity: Option[Int]
)

object LineItem {
  implicit lazy val lineItemJsonFormat: Format[LineItem] = Json.format[LineItem]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/LineItem.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class LineItem 
{
    public string Description { get; set; }
    public string Item { get; set; }
    public int PricePerItem { get; set; }
    public int Total { get; set; }
    public int Tax { get; set; }
    public int Quantity { get; set; }
}


